charts/influxdb3-clustered/values.yaml

```diff
@@ -0,0 +1,222 @@
+  bucket: ""
+
+  # (Required) Provide one of the following (s3, azure, google)
+  # to configure your object store
+#  s3:
+#    # URL for S3 Compatible object store
+#    endpoint: ""
+#
+#    # Set to true to allow communication over HTTP (instead of HTTPS)
+#    allowHttp: "false"
+#
+#    # S3 Access Key
+#    # This can also be provided as a valueFrom: secretKeyRef:
+#    accessKey:
+#      value: ""
+#
+#    # S3 Secret Key
+#    # This can also be provided as a value:
+#    secretKey:
+#      valueFrom:
+#        secretKeyRef: ""
+#
+#    # This value is required for AWS S3, it may not be required for other providers and can be omitted in that case.
+#    region: ""
+#
+#  azure:
+#    # Azure Blob Storage Access Key
+#    # This can also be provided as a value:
+#    accessKey:
+#      valueFrom:
+#        secretKeyRef: "" # azure-access-key
+#
+#    # Azure Blob Storage Account
+#    # This can also be provided as a valueFrom: secretKeyRef:
+#    account:
+#      value: ""
+#
+#  google:
+#    # This section is not needed if you are using GKE Workload Identity. It is only required to use explicit
+#    # service account secrets (JSON files)
+#    serviceAccountSecret:
+#      # Kubernetes Secret name containing the credentials for a Google IAM Service Account.
+#      name: ""
+#      # The key within the Secret containing the credentials.
+#      key: ""
+
+ingesterStorage:
+  # (Optional) Set the storage class. This will differ based on the K8s environment and desired storage characteristics.
+  # If not set, the default storage class will be used.
+  storageClassName: null
+  # Set the storage size (minimum 2Gi recommended)
+  storage: "2Gi"
+
+monitoringStorage:
+  # Set the storage size (minimum 1Gi recommended)
+  storage: "1Gi"
+  # (Optional) Set the storage class. This will differ based on the K8s environment and desired storage characteristics.
+  # If not set, the default storage class will be used.
+  storageClassName: null
+
+# (Optional)
+# We currently only support the ingress NGINX ingress controller: https://github.com/kubernetes/ingress-nginx
+ingress:
+  hosts:
+    # This is the host on which you will access Influxdb 3.0, for both reads and writes
+    - ""
+  # (Optional)
+  # The name of the Kubernetes Secret containing a TLS certificate, this should exist in the same namespace as the Clustered installation.
+  # If you are using cert-manager, enter a name for the Secret it should create.
+  # tlsSecretName: ""
+  http:
+    # Usually you have only one ingress controller installed in a given cluster.
+    # In case you have more than one, you have to specify the "class name" of the ingress controller you want to use
+    className: nginx
+  grpc:
+    # Usually you have only one ingress controller installed in a given cluster.
+    # In case you have more than one, you have to specify the "class name" of the ingress controller you want to use
+    className: nginx
```

Comment:
I wonder whether the default here should be blank as opposed to `nginx`?

I do think this is fine, since the default of the [nginx helm chart also uses the `nginx` class](https://github.com/kubernetes/ingress-nginx/blob/b9e0c281d060ff3030daf300790a04e0fdb89a2e/charts/ingress-nginx/values.yaml#L119-L123) anyway. Plus, `ingress` here is optional.

This is mainly something which came to mind after looking over it again, not a deal breaker I don't think :smile:

Reply:
Yeah, I think that's why I chose nginx as the default. But we could leave them empty, since it's optional.